src/styles/fontSizes.js

```diff
@@ -1,7 +1,9 @@
 // font sizes heading
-export const FONT_SIZE_HEADING_SMALL = '1rem';
-export const FONT_SIZE_HEADING_MEDIUM = '1.25rem';
-export const FONT_SIZE_HEADING_LARGE = '1.5rem';
+export const FONT_SIZE_HEADING_XX_SMALL = '1rem';
+export const FONT_SIZE_HEADING_X_SMALL = '1.25rem';
+export const FONT_SIZE_HEADING_SMALL = '1.375rem';
+export const FONT_SIZE_HEADING_MEDIUM = '1.5rem';
+export const FONT_SIZE_HEADING_LARGE = '1.75rem';
 export const FONT_SIZE_HEADING_X_LARGE = '2rem';
```

Comment:
we should not change the variables styles because it changes all the component using them
<img width="1079" alt="Screen Shot 2021-02-14 at 3 04 39 PM" src="https://user-images.githubusercontent.com/32083556/107889041-00c68900-6ed6-11eb-9625-772cdf90ce4e.png">